### PR TITLE
bgm loop and scoring modified

### PR DESCRIPTION
**Pull request by vaibhavviking**
The background song wasn't looping. I fixed that. Also I have made the scoring more lenient to make the game more addicting.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -128,7 +128,11 @@ function eatFood() {
     updateFoodCell("remove", gameState.food);
     createFood();
     updateFoodCell("add", gameState.food);
-    gameState.score++;
+    if(gameState.score/5 > 2){
+      gameState.score += Math.floor(gameState.score/10);
```
Comment (indremak) [inline index.js]: Great work! This works differently when score is higher than 20. I would suggest to use `Math.ceil()` function and give player 2 points if he have more than 10 points, 3 points if more than 20 and so on. Also in that case, you wouldn't need `if` statement.

Comment (vaibhavviking) [reply index.js]: starting score is 0. floor and ceil both return 0. we can initialize score with 1 and remove if or use if incase score is 0.

Comment (indremak) [reply index.js]: You can add 1 to score before dividing it by 10 and then you won't need if statement.

**Comment by vaibhavviking** [comment]
I have made the required changes. Kindly review them

**Pull request merged**